Rakefile

```diff
@@ -51,6 +51,11 @@ def install_pods_in_example_project()
   $?.exitstatus == 0
 end
 
+def check_pristine()
```

Comment:
The pristine check allows for changes in `CHANGELOG.md` and `UNRELESED.md` as these are expected when doing a release

Reply:
Podspec is done after this point right? So it doesn't have to be checked

Reply:
I don't know what your point is?

Reply:
Do you mean this check should also exclude `Podspec` as it will be checked later anyway?

Reply:
No I think this is fine since `Podspec` is modified later in the flow